sabelotodo/models.py

```diff
@@ -12,7 +17,30 @@ class Item(db.Model):
     order: int = db.Column(db.Integer, nullable=False)
     done: bool = db.Column(db.Boolean, nullable=False)
     description: str = db.Column(db.String)
-    start_date: str = db.Column(db.DateTime)
-    end_date: str = db.Column(db.DateTime)
-    due_date: str = db.Column(db.DateTime)
+    start_date: datetime = db.Column(db.DateTime)
+    end_date: datetime = db.Column(db.DateTime)
+    due_date: datetime = db.Column(db.DateTime)
     parent_id: int = db.Column(db.Integer, db.ForeignKey('items.id'))
+
+class ItemSchema(ma.SQLAlchemyAutoSchema):
+    class Meta:
+        model = Item
+        include_fk = True
+
+    # Override these fields so we can tell Marshmallow more about
+    # time format and time zone. `NaiveDateTime` means we're not 
+    # attempting to store a time zone. The `format` argument means
```

Comment:
Is there a reason to not store a time zone?

Reply:
Honestly, this is just the combination of settings I could finally make work. I wanted to document what they did in case it made sense to change them later.